### PR TITLE
WMS/WMTS now correctly handle Unauthorized response from server

### PR DESCRIPTION
**Pull request by mdastous-bentley**
Implemented the same UI feedback as ArcGIS for WMS/WMTS when a request fails with error 401.

### CODE REVIEW

File: extensions/map-layers/scripts/copy-test-setup.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (calebmshafer) [inline extensions/map-layers/scripts/copy-test-setup.js:1]: You may want to make sure this file and the setup-tests.js are excluded from the npm package.

**Comment by calebmshafer** [comment]
@mdastous-bentley the test setup looks good in the Extension other than the minor comment I left about making sure they aren't included in the npm pkg.

File: core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  WmsMapLayerImageryProvider,
+  WmtsCapabilities,
+  WmtsMapLayerImageryProvider,
+} from "../../../tile/internal";
+
+chai.use(chaiAsPromised);
+
+const wmsSampleSource = { formatId: "WMS", url: "https://localhost/wms", name: "Test WMS" };
+
+describe.only("WmsMapLayerImageryProvider", () => {
```
Comment (pmconne) [inline core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts:26]: remove `.only`.

**Review by pmconne — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

File: core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts (hunk 2 above)
Comment (mdastous-bentley) [reply core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts:26]: nice catch, classic error.

**Pull request merged**